scripts/parse_data_from_cldr.py

```diff
@@ -0,0 +1,143 @@
+"""
+The raw CLDR data was retreived on 25th Jun , 2020 from the following link
+https://github.com/unicode-cldr/cldr-rbnf
+"""
+
+import os
+import json
+import re
+
+SOURCE_PATH = "../number_parser_data/raw_cldr_translation_data/"
+SUPPLEMENTARY_PATH = "../number_parser_data/supplementary_translation_data/"
+TARGET_PATH = "../number_parser/data/"
+
+VALID_KEYS = ["spellout-cardinal", "spellout-numbering"]
+CAPTURE_BRACKET_CONTENT = r'\{(.*?)\}'
+REQUIRED_DATA_POINTS = ["UNIT_NUMBERS", "BASE_NUMBERS", "MTENS", "MHUNDREDS", "MULTIPLIERS", "VALID_TOKENS"]
+
+
+def _is_valid(key):
+    for valid_key in VALID_KEYS:
+        if valid_key in key:
+            return True
+    return False
+
+
+def _parse_base_words(number, word, language_data):
```

Comment:
As the goal of this function is to add new values to the `language_data`, I would rename this to `_add_base_words` to describe its function better. The same for `_parse_compound_words` and `_parse_multiplier_words`.

Reply:
Will update